pandas/core/dtypes/concat.py

```diff
@@ -242,6 +242,72 @@ def union_categoricals(to_union, sort_categories=False, ignore_order=False):
         - sort_categories=True and Categoricals are ordered
     ValueError
         Empty list of categoricals passed
+
+    Notes
+    -----
+
+    To learn more about categories, see 'link
+    <http://pandas.pydata.org/pandas-docs/stable/categorical.html#unioning>__`
+
+    Examples
+    --------
+
+    If you want to combine categoricals that do not necessarily have
+    the same categories, `union_categoricals` will combine a list-like
+    of categoricals. The new categories will be the union of the
+    categories being combined.
+
+    >>> a = pd.Categorical(["b", "c"])
```

Comment:
can you add a `from pandas.api.types import union_categorical`  at the beginning?